types/rdfjs__namespace/Factory.d.ts

```diff
@@ -1,6 +1,15 @@
 import { NamespaceBuilder } from '.';
 
-export default class Factory {
-    static exports: ['namespace'];
-    namespace(baseIRI: string): NamespaceBuilder;
+export interface NamespaceFactory {
+    // eslint-disable-next-line no-unnecessary-generics
+    namespace<TermNames extends string = any>(baseIRI: string): NamespaceBuilder<TermNames>;
```

Comment:
Why exactly do you want to make this a generic? Also why is it defaulted to `any` rather than `string`?